django/forms/boundfield.py

```diff
@@ -143,6 +144,7 @@ def label_tag(self, contents=None, attrs=None, label_suffix=None):
         # Only add the suffix if the label does not end in punctuation.
         # Translators: If found as last label character, these punctuation
         # characters will prevent the default label_suffix to be appended to the label
+        contents = force_text(contents)
```

Comment:
Unless you can include a test showing why `force_text` is needed, use `str` instead. Following the removal of Python 2 support, there are many commits removing `force_text()` usage.

Reply:
OK. I've got some other related work to go in with this, I'll create a new PR.